CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ If your repo has certain guidelines for contribution, put them here ahead of the
 
 - [Contributor License Agreement](https://git.k8s.io/community/CLA.md) Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
 - [Kubernetes Contributor Guide](http://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](http://git.k8s.io/community/contributors/guide#contributing)
-- [Contributor Cheat Sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet.md) - Common resources for existing developers
+- [Contributor Cheat Sheet](https://k8s.dev/cheatsheet) - Common resources for existing developers
```

Comment:
i think the correct link to this would be https://github.com/kubernetes/community/blob/master/contributors/guide/contributor-cheatsheet/README.md

Reply:
@Kartik494 , Bob has suggested me this link.

Reply:
@amolmote got your point , but i have pasted this link because it seems more appropriate , anyways let's wait for other reviewers on this.